crates/ntx-builder/src/state/mod.rs

```diff
@@ -342,14 +342,19 @@ impl State {
             self.in_progress.remove(&prefix);
             tx_impact.account_delta = Some(prefix);
         }
+        // failed to fetch account, so ignore the nullifier
```

Comment:
The comment seems to be on the wrong line.

I would rewrite this loop as follows to make it easier to read:

```rust
        for note in network_notes {
            let prefix = note.account_prefix();

            // Skip and ignore nullifier if note targets a non-existent network account
            let Some(account) = self.fetch_account(prefix).await? else {
                tracing::warn!("could not fetch account from network: {:?}", prefix);
                continue;
            };

            tx_impact.notes.insert(note.nullifier());
            account.add_note(note.clone());
            self.nullifier_idx.insert(note.nullifier(), prefix);
        }
```